pkg/apis/aerogear/v1alpha1/unifiedpushserver_types.go

```diff
@@ -44,6 +42,14 @@ type UnifiedPushServerList struct {
 	Items           []UnifiedPushServer `json:"items"`
 }
 
+type StatusPhase string
+
+var (
```

Comment:
I mostly copied these from the keycloak operator. It had a lot more, that don't apply here (at least not yet). I'm happy to change any of them or add/remove as desired.